stable/nginx-ingress/values.yaml

```diff
@@ -219,6 +219,13 @@ controller:
   #  - name: copy-portal-skins
   #    emptyDir: {}
 
+  initContainers: {}
```

Comment:
Should be an empty list. Please rename it for consistency. 

`extraInitContainers: [] `

Reply:
OK. Just a question - shouldn't be `extraContainers`, `extraVolumes`, etc. lists as well? 

https://github.com/kubernetes/charts/blob/6b619a5bfd0c888df7623d5bc390b516d4b6947b/stable/nginx-ingress/values.yaml#L188